executor/api/kafka/server.go

```diff
@@ -49,6 +60,18 @@ type SeldonKafkaServer struct {
 	ServerUrl      *url.URL
 	Workers        int
 	Log            logr.Logger
+	// KafkaSecurityProtocol string
+}
+
+func getSslElements() *SslKakfa {
+	sslElements := SslKakfa{
+		kafkaSslClientCertFile: util.GetEnv("KAFKA_SSL_CLIENT_CERT_FILE", ""),
+		kafkaSslClientKeyFile:  util.GetEnv("KAFKA_SSL_CLIENT_KEY_FILE", ""),
+		kafkaSslCACertFile:     util.GetEnv("KAFKA_SSL_CA_CERT_FILE", ""),
+		kafkaSecurityProtocol:  util.GetEnv("KAFKA_SECURITY_PROTOCOL", ""),
```

Comment:
:upside_down_face: Minor comment, but the security protocol doesn't really belong with SSL config - if the protocol is `plaintext` for example, none of the SSL options should be used or potentially even read